pkg/virt-api/rest/authorizer.go

```diff
@@ -27,10 +27,10 @@ import (
 	"net/http"
 	"strings"
 
-	restful "github.com/emicklei/go-restful"
-	authorization "k8s.io/api/authorization/v1"
+	"github.com/emicklei/go-restful"
+	authv1 "k8s.io/api/authorization/v1"
```

Comment:
Can you move all refactoring to a separate commit? I think it would be easier to see new functionality.

Reply:
Done